src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(firstNumber) {
+    return function(secondNumber) {
+      count++;
+
+      if (count > 3 && count % 2 === 0) {
```

Comment:
you can use ternary operator here

Reply:
Yes, thsnks, but I'm not really sure that its needed here. Will wait for mentor's decision :)